src/bmf.cpp

```diff
@@ -83,23 +87,25 @@ bool CxadbmfPlayer::xadplayer_load()
   size_t ptr = 0;
   int i;
 
-  if(xad.fmt != BMF)
+  if (xad.fmt != BMF)
     return false;
 
 #ifdef DEBUG
   AdPlug_LogWrite("\nbmf_load():\n\n");
 #endif
   if (tune_size < 6)
     return false;
-  if (!strncmp((char *)&tune[0],"BMF1.2",6))
+  if (!strncmp((char *)tune, "BMF1.2", 6))
   {
     bmf.version = BMF1_2;
     bmf.timer = 70.0f;
+    ptr = 6;
   }
-  else if (!strncmp((char *)&tune[0],"BMF1.1",6))
+  else if (!strncmp((char *)tune, "BMF1.1", 6))
   {
```

Comment:
Same here.  It's probably a matter of taste, but as we're often looking at files in a hex editor when implementing this stuff I do like the added clarity you can get from `[0]`.